docs/start/getting-started/fragments/angular/data-model.md

```diff
@@ -119,69 +119,114 @@ Update `src/tsconfig.app.json` to include the "node" compiler option in *types*:
+Create a file at `types/restaurant.ts` to define a type for the Restaurant:
+
+```ts
+export type Restaurant = {
+  id : string,
+  name : string,
+  description : string,
+  city: string
+}; 
+```
+
+Next, update the class to list all items in the database by running a `ListRestaurants` query when the app starts by implementing [OnInit](https://angular.io/api/core/OnInit) and storing the items in an array:
 
 ```javascript
+import { APIService } from '../API.service';
+import { Restaurant } from './../types/restaurant';
 
+@Component(...)
 export class AppComponent implements OnInit {
-  todos: Array<any>;
+  restaurants: Array<Restaurant>;
 
-  async ngOnInit() {
-    this.apiService.ListTodos().then((evt) => {
-      this.todos = evt.items;
+  constructor(private api: APIService) { }
+
+  ngOnInit() {
+    this.api.ListRestaurants().then(event => {
+      this.restaurants = event.items;
     });
   }
-  //Other code....
 }
 ```
 
 Add the following to your `src/app/app.component.html` to display any of the todos you have added:
 
 ```html
-<ul>
-  <li *ngFor="let item of todos">{{item.name}} - {{item.description}}</li>
-</ul>
+<div *ngFor="let restaurant of restaurants">
+  <div>{{ restaurant.city }}</div>
+  <div>{{ restaurant.name }}</div>
+  <div>{{ restaurant.description }}</div>
+</div>
 ```
 
-Finally, to subscribe to realtime data, update `ngOnInit` to setup a subscription on app start and update the `todos` array when new events are received:
+Finally, to subscribe to realtime data, update `ngOnInit` to setup a subscription on app start and update the `restaurants` array when new events are received:
 
 ```javascript
-import { Component, OnInit } from '@angular/core';
-
+@Component(...)
 export class AppComponent implements OnInit {
-
-  async ngOnInit() {
-    //Other code...
-
-    this.apiService.OnCreateTodoListener.subscribe((evt) => {
-      const data = (evt as any).value.data.onCreateTodo;
-      this.todos = [...this.todos, data];
+  ngOnInit() {
+    //Subscribe to changes
+    this.api.OnCreateRestaurantListener.subscribe( (event: any) => {
+      const newRestaurant = event.value.data.onCreateRestaurant;
+      this.restaurants = [newRestaurant, ...this.restaurants];
     });
   }
+}
 ```
 
 <amplify-callout>The code above imports only the API and PubSub category. To import the entire Amplify library use `import Amplify from 'aws-amplify'`. However, importing only the required categories is recommended as it will greatly reduce the final bundle size.
 </amplify-callout>
 
-After restarting your app using `ng serve` go back to your browser and using dev tools you will see data being stored and retrieved in your backend from the console logs. 
+After restarting your app using `ng serve` go back to your browser and using dev tools you will see data being stored and retrieved in your backend from the console logs. 
```

Comment:
```suggestion
Restart your app by invoking `ng serve`. Now, go back to your browser. In the console logs, you will see data being stored and retrieved in your backend. 
```

^ I'm not an Angular guy, but the wording "dev tools," is that a term of art in the framework? If so, maybe "the Angular dev tools"?

Reply:
These refer to the browser dev tools so are not dependent on Angular.